servers/extensions/physics_server_2d_extension.cpp

```diff
@@ -40,11 +40,15 @@ void PhysicsDirectSpaceState2DExtension::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("is_body_excluded_from_query", "body"), &PhysicsDirectSpaceState2DExtension::is_body_excluded_from_query);
 
 	GDVIRTUAL_BIND(_intersect_ray, "from", "to", "collision_mask", "collide_with_bodies", "collide_with_areas", "hit_from_inside", "result");
-	GDVIRTUAL_BIND(_intersect_point, "position", "canvas_instance_id", "collision_mask", "collide_with_bodies", "collide_with_areas", "results", "max_results");
+	GDVIRTUAL_BIND(_intersect_point, "position", "canvas_instance_id", "check_canvas_instance_id", "collision_mask", "collide_with_bodies", "collide_with_areas", "results", "max_results");
```

Comment:
This should be added at the end IMO or it makes a lot of confusing changes, unless there's some specific reason to add it here

Reply:
It made sense to me for it to be next to the `canvas_instance_id` param. Also, it being put after the return param "results" doesn't feel right.